dask/dataframe/utils.py

```diff
@@ -803,3 +803,33 @@ def assert_max_deps(x, n, eq=True):
         assert max(map(len, dependencies.values())) == n
     else:
         assert max(map(len, dependencies.values())) <= n
+
+
+def valid_divisions(divisions):
+    """ Are the provided divisions valid?
+
+    Examples
+    --------
+    >>> valid_divisions([1, 2, 3])
+    True
+    >>> valid_divisions([3, 2, 1])
+    False
+    >>> valid_divisions([1, 1, 1])
+    False
+    >>> valid_divisions([0, 1, 1])
+    True
+    >>> valid_divisions(123)
+    False
+    """
```

Comment:
Add an example with `[0, float('nan'), 1]` returning False?

Reply:
Thanks for the fail case!